conditions.go

```diff
@@ -79,6 +79,13 @@ func GenerateConditions(ctx Context, md []ConditionMetadata) []models.Condition
 		}
 	}
 
+	afibChance := rand.Intn(100)
+	// 3% Chance of AFib in the general population according to Chris [citation needed]
```

Comment:
Depends on how you want to handle this. Currently we only generate people over 65, so I can just change the 3 to 9. More properly, we could add the birthdate to the generator context and check it when generating the condition.